### PR TITLE
Update dependencies for v2.23

### PR DESCRIPTION
**Pull request by codesome**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -49,35 +49,34 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/alertmanager v0.21.0
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.8.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.14.0
+	github.com/prometheus/common v0.15.0
```
Comment (brian-brazil) [inline go.mod:54]: There's a promtool bugfix in here that you'll want in the changelog.

**Review by brian-brazil — COMMENTED** [review]
:+1:

File: go.mod (hunk 1 above)
Comment (codesome) [reply go.mod:54]: Ok

Comment (codesome) [reply go.mod:54]: What is that fix? Not very obvious to me since `common` does not have a changelog.

Comment (brian-brazil) [reply go.mod:54]: Detecting duplicate label names in exposition.

**Pull request merged**